rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMProxyUserInterface.java

```diff
@@ -291,6 +295,40 @@ public AttributeList getMBeanAttributes(ObjectName name, String[] attributes)
         return this.jmxClient.getConnector().getMBeanServerConnection().getAttributes(name, attributes);
     }
 
+    /**
+     * Set a single JMX attribute of the MBean <code>objectName</code>.
+     * Only integer and string attributes are currently supported, see <code>type</code>.
+     *
+     * @param objectName    the object name of the MBean
+     * @param type          the type of the attribute to set ('int' and 'string' are currently supported)
```

Comment:
Why don't we use an enum instead? How do the other REST endpoints manage types?

Reply:
Actually this is the first/only endpoint that requires types, is enum a better approach?

Reply:
Yes, I believe so. Enums only leave a set of possible values to check (and of possible values to use, from the user point of view), unlike the String approach. Enums converts to String easily, and they are much faster to compare in the switch. And all future REST endpoints that need types will be able to use those in a consistent way.

Reply:
Alright that was quite fun to implement, done in 70aee36b6b33a1819156a0ac078cb09189fd6376.
@paraita yet another exception to the method signature :-\